lib/data/include_argument.cpp

```diff
@@ -95,3 +95,14 @@ bool metashell::data::operator==(const include_argument& a_,
 {
   return a_.type == b_.type && a_.path == b_.path;
 }
+
+bool metashell::data::operator<(const include_argument& a_,
+                                const include_argument& b_)
+{
+  return a_.type < b_.type || (a_.type == b_.type && a_.path < b_.path);
```

Comment:
I like to use `std::tie(a_.type, a_.path) < std::tie(b_.type, b_.path)` for this kind of comparisons. Scales better.